package.json

```diff
@@ -97,6 +97,7 @@
     "bson-ext": "^2.0.5",
     "busboy": "^0.3.1",
     "celebrate": "^15.0.0",
+    "comlink": "^4.3.1",
```

Comment:
as communicated on slack, this is installed in the wrong folder; should be installed in `/frontend`, and that is why chromatic build is failing :P

Reply:
please remove this package even though it's also in /frontend directory haha